src/components/organisms/Header/constants.tsx

```diff
@@ -27,12 +27,6 @@ export const PRODUCTION_LINKS = [
         dataCy: 'points',
         className: 'flex laptop:hidden desktop:flex',
     },
-    {
```

Comment:
we don't want to show the campaign page anymore?

Reply:
Yes, it's Masa's requirement.
https://securedfinance.slack.com/archives/C032N8KUNRZ/p1735090129530299?thread_ts=1734657573.965039&cid=C032N8KUNRZ

Reply:
can you remove this please from links.tsx?
{
        text: 'Campaign',
        href: '/campaign',
        icon: <GiftIcon className='h-5 w-5' />,
        isInternal: true,
    },

Reply:
I removed it!